package.json

```diff
@@ -22,13 +22,15 @@
   "peerDependencies": {
     "grommet": "^2.15.2",
     "grommet-icons": "^4.4.0",
+    "react": "^16.13.1",
     "styled-components": "^5.0.0"
   },
   "devDependencies": {
     "@babel/cli": "^7.1.2",
     "@babel/core": "^7.1.2",
     "@babel/node": "^7.8.4",
     "@babel/preset-env": "^7.1.0",
+    "@babel/preset-react": "^7.12.1",
```

Comment:
Why is this package used as a devDependencies of React is only used as a peer dependency? Can you please provide a little background?

Reply:
React is, and has been, a devDependency as well. Until now, it was extraneous because no React/JSX syntax had been used. However, from the dev side in order to support use of JSX in the theme file, we now need to use React and also instruct Babel how to handle this syntax. `[@babel/preset-react](https://babeljs.io/docs/en/babel-preset-react)` informs Babel how to parse and transform JSX during the build process.